packages/kbn-scout/src/playwright/page_objects/maps_page.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '../fixtures/types';
+
+const RENDER_COMPLETE_SELECTOR = '[data-render-complete="true"]';
+export class MapsPage {
+  constructor(private readonly page: ScoutPage) {}
+
+  async goto() {
+    await this.page.gotoApp('maps');
+    await this.waitForRenderCompletion();
+  }
+
+  async waitForRenderCompletion(selector: string = RENDER_COMPLETE_SELECTOR) {
+    // This is my first attempt at a simple solution for test/functional/services/renderable.ts#waitForRender()
+    await this.page.locator(selector).waitFor();
```

Comment:
We need more or less similar implementation of `waitForRender` to be ported in Scout. I would say this PR is a good candidate to add and test it?

Reply:
Sounds like a great idea!